apps/projects/app/components/Card/Issue.js

```diff
@@ -156,37 +182,59 @@ Issue.propTypes = {
 const StyledIssue = styled.div`
   flex: 1;
   width: 100%;
-  min-width: 600px;
+  /*min-width: 600px;*/
   background: ${theme.contentBackground};
   display: flex;
-  padding-left: 10px;
-  height: 112px;
+  /*padding-left: 10px;*/
+  /*height: 112px;*/
```

Comment:
If these arn't needed can we remove them.

Reply:
Ok.